src/modules/programs/apiLoadSettings.js

```diff
@@ -16,74 +17,90 @@ export const apiLoadProgramSettings = async ({
+                                                } else {
+                                                    summarySettings = `${program.eventsDownload} events per OU`
+                                                }
 
-                                        specificSettingsRows.push(newProgramRow)
+                                                const newProgramRow = {
+                                                    ...program,
+                                                    summarySettings,
+                                                }
+
+                                                specificSettingsRows.push(
+                                                    newProgramRow
+                                                )
+                                            }
+                                        }
                                     }
                                 }
-                            }
 
-                            if (res.value.globalSettings) {
-                                globalSettings = res.value.globalSettings
+                                if (res.value.globalSettings) {
+                                    globalSettings = res.value.globalSettings
 
-                                if (
-                                    globalSettings.settingDownload === GLOBAL ||
-                                    globalSettings.settingDownload ===
-                                        PER_ORG_UNIT
-                                ) {
-                                    globalDefaultValues = GlobalProgramSpecial
-                                } else {
-                                    globalDefaultValues = GlobalProgram
+                                    if (
+                                        globalSettings.settingDownload ===
+                                            GLOBAL ||
+                                        globalSettings.settingDownload ===
+                                            PER_ORG_UNIT
+                                    ) {
+                                        globalDefaultValues = GlobalProgramSpecial
+                                    } else {
+                                        globalDefaultValues = GlobalProgram
+                                    }
                                 }
-                            }
 
-                            return {
-                                settings: res.value,
-                                specificSettings,
-                                programNameList,
-                                programListComplete,
-                                specificSettingsRows,
-                                globalSettings,
-                                globalDefaultValues,
-                            }
-                        })
-                }
-            })
-        }
+                                res.value.globalSettings.disableAll = !hasAuthority
```

Comment:
It's better to treat this as constant, rather than mutating it - so you should create a copy below.